spec/lib/base64.spec.js

```diff
@@ -1,10 +1,11 @@
 "use strict";
 
 describe("base64", () => {
-    var base64;
+    var base64, binaryBuffer;
 
     beforeEach(() => {
-        base64 = require("../../lib/base64")();
+        binaryBuffer = require("../../lib/binary-buffer")();
```

Comment:
I'd prefer a mock to be made instead so I can later use `expect(binaryBuffer.toBuffer).toHaveBeenCalledWith("a string")`.